### PR TITLE
[IMP] fieldservice: Add calendar view + directions to location

### PR DESCRIPTION
**Pull request by osi-scampbell**
Add Calendar view to FSM Orders

Added Location Directions under Instructions Tab

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

**Comment by patrickrwilson** [comment]
@osi-scampbell I installed but don't see the calendar view. I went to dashboard and then orders:
image

**Comment by osi-scampbell** [comment]
Currently Fieldservice_geoengine overwrites the action so you do not see the calendar view. Will make a PR changing fieldservice_geoengine to include calendar as well.

**Comment by patrickrwilson** [comment]
@osi-scampbell the team would like the following fields in this order:
Order #
Stage
Location
Location Phone (if possible)
Contact
Assigned To
Assigned To Phone (if possible)
City
State

I'm not sure if the phone numbers are possible, or if two fields can be on one line, and if not, more fields make the calendar look wonky, so see what you can do and how it looks.

File: fieldservice/models/fsm_order.py [hunk 1]
```diff
@@ -141,12 +142,12 @@ def _onchange_customer_id_location(self):
...
     street2 = fields.Char(related="location_id.street2")
     zip = fields.Char(related="location_id.zip")
-    city = fields.Char(related="location_id.city")
+    city = fields.Char(related="location_id.city", string="City")
     state_name = fields.Char(related="location_id.state_id.name",
                              string='State', ondelete='restrict')
     country_name = fields.Char(related="location_id.country_id.name",
                                string='Country', ondelete='restrict')
-    phone = fields.Char(related="location_id.phone")
+    phone = fields.Char(related="location_id.phone" ,string="Location Phone:")
```
Comment (max3903) [inline fieldservice/models/fsm_order.py]:
```suggestion
    phone = fields.Char(related="location_id.phone" ,string="Location Phone")
```

**Review by patrickrwilson — APPROVED** [review]
Looks good, the views get truncated on the calendar if the appointment isn't long enough which is fine.

**Comment by osi-scampbell** [comment]
@max3903 @patrickrwilson I added Location Direction field for the ticket made about this issue: https://github.com/OCA/field-service/issues/220

**Comment by patrickrwilson** [comment]
@osi-scampbell Field looks good, i couldn't figure why i couldn't add a comment in the code but i think the copy_notes just needs to be updated to update that new field instead of the directions. that way your field populates instead of directions, unless there is more to it and you were just prepping this for that change?

**Comment by osi-scampbell** [comment]
@max3903 I'm getting 404 Error on Runbot after my last commit?

**Comment by max3903** [comment]
@osi-scampbell Try removing the sequence completely.

**Comment by osi-scampbell** [comment]
@max3903 Runbot appears to be back online, only 1 warning left, 'Transient module states were reset' in \fieldservice_geoengine\views\fsm_order.xml. It throws a Key Error for 'fieldservice_geoengine.field_fsm_order__shape'

**Pull request merged**